content/cloud-hosting/azure/azure-container-instances.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Azure Container Instances
+showMetadata: true
+editable: true
+showToc: true
+---
+
+# What is Azure Container Instances (ACI)?
+
+Azure Container Instances (ACI) is a managed service that allows you to run containers directly on the Microsoft Azure public cloud, without requiring the use of virtual machines (VMs).
+
+# Azure Container Instance Features
+
+- [Support for both Linux and Windows containers](https://docs.microsoft.com/en-us/azure/container-instances/)
+- [Support for persistent storage by mounting Azure file shares to the container](https://docs.microsoft.com/en-us/azure/container-instances/container-instances-volume-azure-files)
+- [Bursting from AKS to ACI with the Virtual Kubelet](https://docs.microsoft.com/en-us/azure/aks/virtual-nodes-cli)
+
+# Quick Start Azure Container Instances (ACI) using Docker Context and Docker Compose
+
+## Prerequisites
+- [Compose v2](https://github.com/docker/compose-cli)
```

Comment:
👍 Curiosity, can we use with docker-compose v1?

Reply:
Can’t as it does not support docker contexts for ACI and ECS